package.json

```diff
@@ -95,7 +106,8 @@
   "jupyterlab": {
     "extension": "lib/index.js",
     "schemaDir": "schema",
-    "outputDir": "jupyterlab_latex/labextension"
+    "outputDir": "jupyterlab_latex/labextension",
+    "webpackConfig": "./webpack.config.js"
```

Comment:
let's go ahead and also make `pdfjs-dist` a `sharedPackages`... we really don't want more than one of these sneaking around on the page.

Reply:
Done. And here my choices for the shared package config:

`"bundled": true` - share with other extensions
`"singleton": true` - this made sense to me to minimize multiple simultaneous versions (but feel free to correct me here)
`"strictVersion": true` - to make sure only the specified pinned version is used (there are issues with newer pdfjs)